### DISCUSSION TITLE
[RFC] Classification Summaries

### DISCUSSION THREAD

**Discussion by rogerhutchings** [zooniverse/front-end-monorepo#2425] [Ideas]
We need to decide on the the behaviour for classification summaries. Currently, there's an experimental feature to disable them; several people have asked for them to be disabled by default, and a checkbox to enable them added to the lab.

**Comment by eatyourgreens** [comment]
I'd say disabled by default. +1000 to this from @mrniaboc

**Comment by srallen** [comment]
I think there's also a greater question of the utility of generic classification summaries. Maybe it can be deprecated entirely and only show feedback if that is enabled.

**Comment by rogerhutchings** [comment]
This is recommended for deprecation - @chrislintott @trouille how would you feel about this?

**Comment by chrislintott** [comment]
I think it's not worked, and I'd be ok with it going. Its original purpose was the make sure people had the chance to go to Talk, so I'd like a broader conversation about how else we might do that, but this is ok.

**Comment by eatyourgreens** [comment]
From conversations on NfN, and similar transcription projects where the task is quite complicated, they'd be in favour of a summary that shows your entire classification before you submit it. The idea being you can double-check your work to make sure it's ok. Using Back to go back through your classification, then Next to return to the end and submit, is clunky and prone to error.

I think that would be an enhancement as part of broader transcription work, because I don't think the existing summary works very well for that, even if it we showed it before you press Submit.

**Comment by srallen** [comment]
Right, I think some kind of view that can be toggled to display as you are annotating that shows an in-progress summary of annotations would be useful for transcription, but the current version of the generic summary that just displays the JSON of the annotation isn't really useful for that use case. @snblickhan do you have any thoughts about this? I think we should consider this a separate enhancement feature.

It just occurred to me that reviewing the in-progress annotations is why the "persist annotations" feature is often enabled for transcription projects (and they're the project types that mostly use the persistence feature). An in-progress summary might be a UI/UX refactor of that feature?

As for getting to Talk, the zoominar with @beckyrother this week was all about how we can improve this experience when we don't have summaries. A recording of the discussion is here: https://zooniverse.slack.com/archives/C19H6QRMJ/p1573661640018900

**Comment by eatyourgreens** [comment]
@srallen I think you're right. With drawing tasks and survey tasks, all of our previous annotations are visible on the subject while we work through a classification but there's no equivalent for text tasks.

**Comment by beckyrother** [comment]
@chrislintott happy to recap the zoominar this afternoon as well
